src/mappings/Reputation.ts

```diff
@@ -0,0 +1,31 @@
+import 'allocator/arena'
+export { allocate_memory }
+
+import { Entity, Address, Value, store, crypto, ByteArray } from '@graphprotocol/graph-ts'
+
+import { Reputation, Mint, Burn } from '../types/Reputation/Reputation'
+import { concat } from '../utils'
+
+let avatar = Address.fromString('a3f5411cfc9eee0dd108bf0d07433b6dd99037f1')
```

Comment:
why it is hard coded ?

Reply:
Ideally this address should dynamically be replaced by the the address of the avatar whose native reputation was specified in yaml. But since we did not have any auto build currently I had a hard coded address of the avatar corresponding to the reputation contract specified in the yaml.
I'm still unsure how this would fit for a base cache layer